packages/frames.js/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "lint": "eslint \"./src/**/*.{ts,tsx}\"",
-    "build": "NODE_OPTIONS='--max-old-space-size=16384' rimraf ./dist && tsup `find ./src -type f \\( -regex '.*\\.tsx*' -a -not -regex '.*\\.test\\.tsx*' \\)` --format cjs,esm --dts",
+    "build": "NODE_OPTIONS='--max-old-space-size=16384' tsup `find ./src -type f \\( -regex '.*\\.tsx*' -a -not -regex '.*\\.test\\.tsx*' \\)` --format cjs,esm --dts",
```

Comment:
could we at least remove `dist` when publishing a package? First remove dist, then build and then publish? The idea is that `dist` should never contain any artifacts that don't exist in source.

Reply:
Ok this is not necessary since tsup cleans `dist`.

Reply:
okay great